<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoButtonSinglePPL</class>
 <widget class="QWidget" name="PhotoButtonSinglePPL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>170</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label_photo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>168</width>
        <height>214</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>214</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>168</width>
        <height>214</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../bordingcontrol_linux.qrc">:/3实时登机/Images/3实时登机/3.2.1/照片-小.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_transferType">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../bordingcontrol_linux.qrc">:/3实时登机/Images/3实时登机/3.2.1/过站.png</pixmap>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_info" native="true">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_info{border-radius:10px;background-color: rgb(1, 65, 109);border-image: url(:/3实时登机/Images/3实时登机/3.2.1/矩形-4363.png);}


</string>
     </property>
     <widget class="QLabel" name="label_seatNO">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>82</width>
        <height>54</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>OINF</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_flightDay">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>15</y>
        <width>90</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>2020.05.05</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bordingcontrol_linux.qrc"/>
 </resources>
 <connections/>
</ui>
